rd_ui/app/scripts/directives/plotly.js

```diff
@@ -45,59 +45,82 @@
     });
   };
 
-  var normalAreaStacking = function(seriesList) {
-    fillXValues(seriesList);
+  var storeOriginalHeightForEachSeries = function(seriesList) {
+    _.each(seriesList, function(series) {
+      if(!_.has(series,'visible')){
+        series.visible = true;
+        series.original_y = series.y.slice();
+      }
+    });
+  };
+
+  var getEnabledSeries = function(seriesList){
+    return _.filter(seriesList, function(series) {
```

Comment:
Actually, this would not be the same. Because of Javascript truthiness, when `series.visible = 'legendonly'` would evaluate to `true`.